README.md

```diff
@@ -80,100 +71,95 @@ Now we'll convert it using HTML5 Semantic Elements
 </div>
 ```
 
-Notice that in cases where the content within the element is not semantically related or we have the need to simply create a generic box such as the `wrapper`, we can still use divs as we please.
+Notice that in cases where the content within the element is not semantically related or we have the need to create a generic box such as the `wrapper`, we can still use `div` elements as we please.
+
+## Identify Specific HTML5 Semantic Elements
 
-### Some Semantic Elements Worth Knowing
+Now that we have so many semantic elements we can use in our HTML pages, let's cover the most common and useful ones.
 
-Next let's talk about the semantic meaning intended for each element so you have an idea of how they were meant to be used. As a developer there are no hard rules of how to configure these elements. We only have their intended meaning to work off of, so you should feel comfortable configuring them in a way that makes the most sense to you and best suits your layout needs. The following descriptions are as they are stated from Mozilla Developer Networks documentation.
+As a developer, keep in mind that while these elements are intended for certain content, there are no hard rules about how to configure them. You should feel comfortable configuring them in any way that makes the most sense to you and best suits your layout needs.
 
-#### Article
+The following descriptions come from the documentation at the Mozilla Developer Network.
+
+### Article
 
 `<article>...</article>`
 
 The HTML `<article>` element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). This could be a forum post, a magazine or newspaper article, a blog entry, an object, or any other independent item of content. Each `<article>` should be identified, typically by including a heading (`<h1>-<h6>` element) as a child of the `<article>` element.
 
-#### Aside
+### Aside
 
 `<aside>...</aside>`
 
 The HTML `<aside>` element represents a section of the page with content connected tangentially to the rest, which could be considered separate from that content. These sections are often represented as sidebars or inserts. They often contain the definitions on the sidebars, such as definitions from the glossary; there may also be other types of information, such as related advertisements; the biography of the author; web applications; profile information or related links on the blog.
 
-#### Details
+### Details
 
 `<details>..</details>`
 
 The HTML Details Element (`<details>`) is used as a disclosure widget from which the user can retrieve additional information.
 
-#### Figure & Figcaption
+### Figure & Figcaption
 
```

Comment:
This would be a good one for a code and render example (i.e. show the HTML code, then show them what would be rendered with that code).